src/server.js

```diff
@@ -1,26 +1,16 @@
 const express = require('express');
 const cors = require('cors');
 const dotenv = require('dotenv');
-const tokenHandler = require('./handlers/token');
 
 dotenv.config();
 
 const port = process.env.PORT;
-const oauth2Endpoint = process.env.VERTEX_OAUTH2_ENDPOINT;
-const clientId = process.env.VERTEX_CLIENT_ID;
-const clientSecret = process.env.VERTEX_CLIENT_SECRET;
-const model = process.env.DEFAULT_MODEL;
-const expiresInSeconds = 2 * 60 * 60; // Duration for how long a token is valid for. Set to 2 hours.
+const streamKey = process.env.DEFAULT_STREAM_KEY;
 
 const app = express();
 app.use(cors());
 
-app.get(
-  '/token',
-  tokenHandler(oauth2Endpoint, clientId, clientSecret, expiresInSeconds)
-);
-
-app.get('/model', (_, res) => res.json({ urn: model }));
+app.get('/stream-key', (_, res) => res.json({ key: streamKey }));
```

Comment:
At this point, I'm not sure if we even need a Node backend. There's no communication back to Vertex.

Reply:
Yeah, it currently isn't really necessary with these changes. I can remove it for now, and we can always add it back later if it ends up being needed